policy-engine/src/graph.rs

```diff
@@ -12,19 +12,20 @@ use opentelemetry::{
     trace::{mark_span_as_active, FutureExt, Tracer},
     Context as ot_context,
 };
-use prometheus::{histogram_opts, Counter, Histogram, Registry};
+use prometheus::{histogram_opts, Histogram, IntCounterVec, Opts, Registry};
 use serde_json;
 use std::collections::HashMap;
 
 lazy_static! {
-    static ref V1_GRAPH_INCOMING_REQS: Counter = Counter::new(
-        "v1_graph_incoming_requests_total",
-        "Total number of incoming HTTP client request to /graph"
+    static ref GRAPH_INCOMING_REQS: IntCounterVec = IntCounterVec::new(
+        Opts::new("graph_incoming_requests_total",
+        "Total number of incoming HTTP client request to /graph"),
```

Comment:
`to /graph`, like [this thread][1].

[1]: https://github.com/openshift/cincinnati/pull/537#discussion_r718784762